app/models/collection_item.rb

```diff
@@ -145,14 +149,33 @@ def approve_automatically
 
         users.each do |user|
           if user.preference.automatically_approve_collections || (collection && collection.user_is_posting_participant?(user))
+            # if the work is being added by a collection maintainer and at
+            # least ONE of the works owners allows automatic inclusion in collections,
+            # add the work to the collection
             approve_by_user
+            users.each do |email_user|
+              UserMailer.added_to_collection_notification(email_user.id, item.id, collection.id).deliver!
+            end
             break
           end
         end
       end
     end
   end
-  
+
+  before_save :send_work_invitation
+  def send_work_invitation
```

Comment:
Assignment Branch Condition size for send_work_invitation is too high. [18.44/15]